From: Jiri Slaby <jslaby@suse.cz>
Subject: disable -pedantic
Patch-mainline: no

-pedantic generates 3623 warnings at the moment. Disable it for now, so
that we can see more important warnings.
---
 CMake/unix_config.cmake |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/CMake/unix_config.cmake
+++ b/CMake/unix_config.cmake
@@ -2,8 +2,8 @@ message(STATUS "Setting Unix configurati
 
 macro(os_set_flags)
     set(CMAKE_POSITION_INDEPENDENT_CODE ON)
-    set(CMAKE_C_FLAGS   "${CMAKE_C_FLAGS}   -pedantic -g -D_DEFAULT_SOURCE")
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -pedantic -g -Wno-missing-field-initializers")
+    set(CMAKE_C_FLAGS   "${CMAKE_C_FLAGS}   -g -D_DEFAULT_SOURCE")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -g -Wno-missing-field-initializers")
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-switch -Wno-multichar -Wsequence-point -Wformat -Wformat-security")
 
     execute_process(COMMAND ${CMAKE_C_COMPILER} -dumpmachine OUTPUT_VARIABLE MACHINE)
